.github/workflows/emscripten.yml

```diff
@@ -36,86 +33,36 @@ concurrency:
   cancel-in-progress: true
 
 permissions:
-  contents: read # to fetch code (actions/checkout)
+  contents: read  # to fetch code (actions/checkout)
 
 jobs:
   build-wasm-emscripten:
     name: Build NumPy distribution for Pyodide
     runs-on: ubuntu-22.04
     # To enable this workflow on a fork, comment out:
     if: github.repository == 'numpy/numpy'
-    env:
-      PYODIDE_VERSION: 0.26.0
```

Comment:
I feel that it would be better not to pin the Pyodide version for this workflow. Two reasons:

1. The `pyodide-build` version is coupled with the Pyodide version, which in-turn is set by `cibuildwheel`. In my earlier commits, I used `pip` to install `pyodide-build`, which can fall out of date
2. NumPy is one of the fundamental packages and is extensively built and tested with Pyodide when the Emscripten toolchain version is bumped (i.e., when the ABI breaks). It's quite unlikely that building it would be broken for a released version of Pyodide, and owing to the experimental nature of the build system and this feature in general, `cibuildwheel` will not plan to provide WebAssembly builds for a tip-of-tree or unlisted Pyodide build – which would require its own CDN or distribution channel, anyway.

Do you have any thoughts about this, @rgommers?